meta-spec-0000.md

```diff
@@ -95,7 +94,7 @@ However, we highly recommended that you first do one or more of the following:
 
 To get feedback from the larger community, we also recommend creating a new
 [discussion topic](https://discuss.scientific-python.org/c/specs/ideas/9)—under
-the `Accepted` subcategory of the `SPECs`
+the `Proposed` subcategory of the `SPECs`
```

Comment:
I realize the terminology here isn't settled yet, but I thought asking potential SPEC authors to open a discussion in the `Accepted` category might be confusing since the term "Accepted" already has special meaning (i.e. accepted by the SSC). I realize there is no "Proposed" category yet, but I am just throwing something out there as a potential name to distinguish from the "Accepted" discussion.

Re: organizing categories on discuss.scientfic-python.org, maybe the following category names (or similar) could be used:
 - `Ideas` - for people who have an idea and want to have preliminary discussion to see if it's a good fit for a SPEC
 - `Proposed` - for discussion of draft SPECs, i.e. actual SPECs for review by the SSC and wider community
 - `Active` - for questions/comments/feedback about SPECs that have been accepted by the SSC

Note that I'm mostly thinking about how discussions can be organized. The main thing for me is to make sure there is a distinction between SPECs that have been accepted by the SSC and SPECs that are still in the proposal phase.

Reply:
I was imaging that ideas and proposed SPECs would both go under the `Ideas` category on discourse.  And that once a SPEC was accepted, it would be given a number, merged to master, and have a `Accepted` topic on discourse.  This looks like it is partially leftover from an earlier edit.  I will review and fix this.